### PR TITLE
APP-63 Fix event demo

### PR DESCRIPTION
**Pull request by ohamov**

### CODE REVIEW

File: eventdemo/source/c/CMakeLists.txt [hunk 1]
```diff
@@ -15,23 +15,22 @@
...
+set(APP_NAME "demo_client")
+set(KAA_SDK_DIR "client-c")
 
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/src
-                    ${CMAKE_CURRENT_SOURCE_DIR}/libs/kaa/src
-                   )
+execute_process(COMMAND ${CMAKE_COMMAND} -E make_directory ${KAA_SDK_DIR})
+execute_process(COMMAND sh -c "tar -zxf ${CMAKE_CURRENT_SOURCE_DIR}/libs/kaa/kaa-c*.tar.gz -C ${KAA_SDK_DIR}")
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu99 -g -Wall -Wextra")
```
Comment (rasendubi) [inline eventdemo/source/c/CMakeLists.txt]: Change std to `c99`, please.

File: eventdemo/source/c/build.sh [hunk 2]
```diff
@@ -15,10 +15,12 @@
 #  limitations under the License.
 #
 
+# Exits immediately if error occurs
+set -e
 
 RUN_DIR=`pwd`
```
Comment (rasendubi) [inline eventdemo/source/c/build.sh]: Replace `RUN_DIR` with `PROJECT_HOME`, please. They are same

**Comment by forGGe** [comment]
Looks good

**Comment by rasendubi** [comment]
LGTM

**Pull request merged**